checkpoints/bulk_rna_bert_gtex_encode/config.json

```diff
@@ -0,0 +1 @@
+{"n_genes": 19062, "n_expressions_bins": 64, "embed_dim": 256, "init_gene_embed_dim": 200, "project_gene_embedding": true, "use_gene_embedding": true, "num_attention_heads": 8, "key_size": 32, "ffn_embed_dim": 512, "num_layers": 4, "use_memory_efficient_attention": false, "use_gradient_checkpointing": true, "gene2vec_weights_path": "data/gene2vec_weights_common_gene_ids.npy", "embeddings_layers_to_save": [], "attention_layers_to_save": [], "rnaseq_tokenizer_bins": [0.0, 0.015873015873015872, 0.031746031746031744, 0.047619047619047616, 0.06349206349206349, 0.07936507936507936, 0.09523809523809523, 0.1111111111111111, 0.12698412698412698, 0.14285714285714285, 0.15873015873015872, 0.1746031746031746, 0.19047619047619047, 0.20634920634920634, 0.2222222222222222, 0.23809523809523808, 0.25396825396825395, 0.2698412698412698, 0.2857142857142857, 0.30158730158730157, 0.31746031746031744, 0.3333333333333333, 0.3492063492063492, 0.36507936507936506, 0.38095238095238093, 0.3968253968253968, 0.4126984126984127, 0.42857142857142855, 0.4444444444444444, 0.4603174603174603, 0.47619047619047616, 0.49206349206349204, 0.5079365079365079, 0.5238095238095237, 0.5396825396825397, 0.5555555555555556, 0.5714285714285714, 0.5873015873015872, 0.6031746031746031, 0.6190476190476191, 0.6349206349206349, 0.6507936507936507, 0.6666666666666666, 0.6825396825396826, 0.6984126984126984, 0.7142857142857142, 0.7301587301587301, 0.746031746031746, 0.7619047619047619, 0.7777777777777777, 0.7936507936507936, 0.8095238095238095, 0.8253968253968254, 0.8412698412698412, 0.8571428571428571, 0.873015873015873, 0.8888888888888888, 0.9047619047619047, 0.9206349206349206, 0.9365079365079365, 0.9523809523809523, 0.9682539682539681, 0.9841269841269841, 1.0], "use_log_normalization": true, "use_max_normalization": true, "normalization_factor": 5.873553674586928}
```

Comment:
Could it make sense to re-factor the RNA-seq tokenizer to take `num_bins` as an argument rather than hard-coding bins in json files ?

Reply:
The binning also depends on the range of the RNA-seq TPMs (which is determined by the pre-training dataset of the language model). In this first approach, we decided to re-scale all TPMs between 0 and 1 (with `use_max_normalization=True`), but that may not be always the case, thus only `num_bins` might not be enough as an argument?